javascript/graphql-apollo-server/README.md

```diff
@@ -173,7 +173,7 @@ The first step would be to add a new table, e.g. called `Profile`, to the databa
 CREATE TABLE "Profile" (
   "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
   "bio" TEXT,
-  "user" TEXT NOT NULL UNIQUE REFERENCES "User"(id) ON DELETE SET NULL
+  "user" INTEGER NOT NULL UNIQUE REFERENCES "User"(id) ON DELETE SET NULL
```

Comment:
Intentional change?

Reply:
Not me... I don't know how this changed?

Reply:
Hmm weird... using `INTEGER` here is correct however 🤔🤔🤔